frontend/src/models/Garage.model.ts

```diff
@@ -3,9 +3,22 @@ import { systemClient } from '../services/System';
 import { saveAsJson } from '../utils';
 export interface Slot {
   robot: Robot;
+  lastOrderId: number | null;
+  lastOrderShortAlias: string | null;
+  activeOrderId: number | null;
+  activeOrderShortAlias: string | null;
   order: Order | null;
 }
```

Comment:
This is the most important change, everything else is just adapting the code to it